### PR TITLE
Add makefile target to pull images from CI and push to quay

### PR DESCRIPTION
**Pull request by chmouel**
Add makefile target instead of homegrown shell script

/cc @arilivigni @sthaha @vdemeester @hrishin

### CODE REVIEW

**Comment by chmouel** [comment]
I am having a problem with imagedigestexporter, the job here say it's pushed : 

https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/openshift_tektoncd-pipeline/37/pull-ci-openshift-tektoncd-pipeline-release-v0.4.0-e2e/4

`
operator Build image tektoncd-pipeline-imagedigestexporter from the repository 
`

but i can't find it in `registry.svc.ci.openshift` ie : 

image

am I missing something ? (URL changes?)

same goes for gsutil I am not sure why we make an image but we don't publish it ? is that obsolete?

**Comment by chmouel** [comment]
the right URL is  `registry.svc.ci.openshift.org/openshift/tektoncd-v0.4.0:tektoncd-pipeline-` was adjusted (tks @vdemeester ) and this has been pulled from ci/pushed to quay properly : 

image

File: Makefile [hunk 1]
```diff
@@ -5,6 +5,14 @@ GOOS=linux
 CORE_IMAGES=./cmd/bash ./cmd/controller ./cmd/entrypoint ./cmd/gsutil ./cmd/kubeconfigwriter ./cmd/nop ./cmd/webhook ./cmd/imagedigestexporter
 CORE_IMAGES_WITH_GIT=./cmd/creds-init ./cmd/git-init
 
+##
+# You need to provide a RELEASE_VERSION when using targets like `push-image`, you can do it directly
+# on the command like this: `make push-image RELEASE_VERSION=0.4.0`
+RELEASE_VERSION=
+REGISTRY_CI_URL=registry.svc.ci.openshift.org/openshift/tektoncd-v$(RELEASE_VERSION):tektoncd-pipeline
+REGISTRY_RELEASE_URL=quay.io/openshift-pipeline/tektoncd-pipeline
+BINARY_WITHOUT_IMAGE=gsutil # TODO: why those are not in CI ?
```
Comment (vdemeester) [inline Makefile]: This won't be needed anymore with #40

Comment (chmouel) [reply Makefile]: But is the pushing image from private ci to public quay needed in the future?

**Comment by chmouel** [comment]
Pushed!! 🕺 

```bash
% make push-image RELEASE_VERSION=0.4.0
+ for image in ./cmd/bash ./cmd/controller ./cmd/entrypoint ./cmd/gsutil ./cmd/kubeconfigwriter ./cmd/nop ./cmd/webhook ./cmd/imagedigestexporter
++ basename ./cmd/bash
+ image=bash
[...]
+ docker pull registry.svc.ci.openshift.org/openshift/tektoncd-v0.4.0:tektoncd-pipeline-bash
tektoncd-pipeline-bash: Pulling from openshift/tektoncd-v0.4.0
c2340472a0fa: Already exists
6e55351c18ff: Already exists
b2d2704dda6c: Already exists
a2c10be042b9: Already exists
3a128decb18a: Pull complete
[....]
```

**Pull request merged**